frontend/src/App.js

```diff
@@ -0,0 +1,13 @@
+import { render } from "react-dom";
+import GuessInput from "./GuessInput";
+
+const App = () => {
+  return (
+    <div>
+      <h2>Try to the guess the wordle of the day!</h2>
+      <GuessInput />
```

Comment:
Yea, this is a great suggestion.  I'd like to try your second suggestion as it would make things more re-usable like you say, but I may need a little guidance on this one if you don't mind chatting about it.

- [ ] Discuss refactoring the GuessInput component to be more re-usable / accurate